### PR TITLE
check logging namespace for actual resources

### PR DESCRIPTION
**Pull request by jcantrill**
3.9 cherry-pick of #1067

### CODE REVIEW

File: test/access_control.sh [hunk 1]
```diff
@@ -264,9 +264,11 @@ fi
...
 os::log::info now auth using admin + token
 get_test_user_token $LOG_ADMIN_USER $LOG_ADMIN_PW
-nrecs=$( curl_es_with_token $espod "/${logging_index}/_count" $test_name $test_token | \
-         get_count_from_json )
-os::cmd::expect_success "test $nrecs -gt 1"
+if [ ${LOGGING_NS} = "logging" ] && [ $espod != $esopspod] ; then
+  nrecs=$( curl_es_with_token $espod "/${logging_index}/_count" $test_name $test_token | \
+           get_count_from_json )
+  os::cmd::expect_success "test $nrecs -gt 1"
+fi
```
Comment (richm) [inline test/access_control.sh]: What about checking for `project.*` then?
```
nexpectedrecs=$( curl_es $espod "/project.*/_count" | get_count_from_json )
nactualrecs=$( curl_es_with_token $espod "/project.*/_count" $test_name $test_token | get_count_from_json )
os::cmd::expect_success "test $nexpectedrecs -gt $nactualrecs"
```

File: hack/testing/util.sh [hunk 2]
```diff
@@ -195,7 +195,7 @@ function wait_for_fluentd_to_catch_up() {
     local uuid_es_ops=$( uuidgen | sed 's/[-]//g' )
     local expected=${3:-1}
     local timeout=${TIMEOUT:-600}
-    local project=${4:-logging}
+    local project=${4:-openshift-logging}
```
Comment (nhosoi) [inline hack/testing/util.sh]: It's better to use ${LOGGING_NS}?
local project=${4:-${LOGGING_NS}}

**Comment by nhosoi** [comment]
Regarding the mux test failure, I think this line "oc exec $muxpod -- cat /var/log/fluentd.log > $ARTIFACT_DIR/mux.mux.pod.int.log" could be just removed as follows.
--- a/test/mux.sh
+++ b/test/mux.sh
@@ -354,7 +354,6 @@ cleanup() {
             oc get configmap/logging-mux -o yaml > $ARTIFACT_DIR/mux.mux.configmap.yaml
             oc exec $muxpod -- ls -alrtF /etc/fluent/configs.d/openshift 2>&1 | artifact_out
             oc exec $muxpod -- ls -alrtF /etc/fluent/configs.d/user 2>&1 | artifact_out
-            oc exec $muxpod -- cat /var/log/fluentd.log > $ARTIFACT_DIR/mux.mux.pod.int.log
         fi
     fi
     $mycmd mux test finished at $( date )

**Pull request closed without merging**